README.md

```diff
@@ -47,19 +47,24 @@ yarn deploy-local
   block(id: "5200793") {
     id
     header {
+      id
```

Comment:
I would remove the ids from the query. They will all be the same (block hash if I remember correctly, or height maybe), they are irrelevant for testing/checking purpose and will make the query output much larger.

Reply:
Yes that true, maybe output seems to be a little bit larger but if we save empty entities then it's easier to catch it, for example empty array. What do you think?